src/crimson/os/seastore/async_cleaner.cc

```diff
@@ -1336,7 +1377,7 @@ AsyncCleaner::reserve_projected_usage(std::size_t projected_usage)
   if (disable_trim) {
     return seastar::now();
   }
-  ceph_assert(init_complete);
+  ceph_assert(state >= cleaner_state_t::READY);
```

Comment:
Probably worth adding an is_ready() helper for this.

Reply:
done